content/en/apps/guides/performance/telemetry.md

```diff
@@ -39,8 +39,13 @@ The telemetry data gathered changes with different versions of the framework. Cu
 | `boot_time` | The overall boot time including loading the code, purging, and accessing the database. |
 | `boot_time:1:to_first_code_execution` | The time between the page loading and the JavaScript starting to run. |
 | `boot_time:2:to_bootstrap` | The time between JavaScript starting and the bootstrapping (purging, initial replication, etc) to complete. |
+| `boot_time:2_1:to_replication` | The time it takes to complete the replication. Added in 3.14. |
+| `boot_time:2_2:to_purge` | The time it takes to complete the DB purge. Added in 3.14. |
+| `boot_time:2_3:to_purge_meta` | The time it takes to complete the Meta purge. Added in 3.14. |
 | `boot_time:3:to_angular_bootstrap` | The time between bootstrapping completing and the webapp being ready to use. |
 | `boot_time:4:to_db_warmed` | The time between the webapp being ready to use and the database being ready to use. |
+| `boot_time:purgedDB:<boolean>` | `boot_time:purgedDB:true` if DB was purged, `boot_time:purgedDB:false` if DB wasn't purged. Added in 3.14. |
```

Comment:
I added a comment in the other PR about changing the name of this entry. Also, I think saying `DB` is a bit too abstract. What is DB? 
Maybe:
```suggestion
| `boot_time:purging:<boolean>` | `boot_time:purging:true` when purging executed successfully on device startup, `boot_time:purging:false` when purging did not run. Added in 3.14. |
```

I got the idea that maybe this should be a binary where purging had to run and succeeded, or didn't have to run, and have another entry when purging had to run but failed.